### DISCUSSION TITLE
Current Usage - how to reduce

### DISCUSSION THREAD

**Discussion by tt5552** [getsentry/support#17] [Support] [answered]
Hi,

An existing and persistent issue activated the Spike protection because it is frequent.

1. I selected 'Ignore' on it, does this mean sentry will now not count any events of this issue toward my existing sentry plan?

2. What happens if I go above monthly data plan, do I have to pay or does it stop reporting issues?

Thanks.

**Comment by markstory** [comment, marked as answer]
> I selected 'Ignore' on it, does this mean sentry will now not count any events of this issue toward my existing sentry plan?

No, ignoring a issue only suppresses notifications and alerts about that issue.

> What happens if I go above monthly data plan, do I have to pay or does it stop reporting issues?

Once your usage goes over your reserved quotas, your organization will start to use on-demand budgets if configured. Once all the on-demand budget has been used, future errors will not be processed.
